lmms_eval/tasks/clotho_aqa/utils.py

```diff
@@ -49,9 +49,13 @@ def clotho_aqa_doc_to_text(doc, lmms_eval_specific_kwargs):
         "Authorization": f"Bearer {API_KEY}",
         "Content-Type": "application/json",
     }
+
 elif API_TYPE == "azure":
-    API_URL = os.getenv("AZURE_ENDPOINT", "https://api.cognitive.microsoft.com/sts/v1.0/issueToken")
-    API_KEY = os.getenv("AZURE_API_KEY", "YOUR_API_KEY")
+    API_KEY = os.getenv("AZURE_API_KEY", None)
+    deployment_name = os.getenv("AZURE_DEPLOYMENT_NAME", None)
+    resource_name = os.getenv("AZURE_RESOURCE_NAME", None)
+    api_version = os.getenv("AZURE_API_VERSION", None)
+    API_URL = f"https://{resource_name}.openai.azure.com/openai/deployments/{deployment_name}/chat/completions?api-version={api_version}"
```

Comment:
A small question on how we format this azure api url. Why don't we simple export the formatted to `API_URL` and then use the original format. This keeps your env clean and less export operation needed.